### PR TITLE
bump helm version > 2.17.0

### PR DESCRIPTION
**Pull request by neilpeterson**
Fixes https://github.com/mspnp/microservices-reference-implementation/issues/205.

@doodlemania2 / @ferantivero

### CODE REVIEW

**Comment by neilpeterson** [comment]
I've added to this PR to also fix https://github.com/mspnp/microservices-reference-implementation/issues/207

**Comment by neilpeterson** [comment]
I added to this PR an update to several outdated az commands.

File: deployment.md [hunk 1]
```diff
@@ -179,7 +179,7 @@ Note: the tested nmi version was 1.4. It enables namespaced pod identity.
 
 ```bash
 # setup AAD pod identity
-helm install aad-pod-identity/aad-pod-identity --set=installCRDs=true --set nmi.allowNetworkPluginKubenet=true --name aad-pod-identity --namespace kube-system
+helm install aad-pod-identity/aad-pod-identity --set=installCRDs=true --set nmi.allowNetworkPluginKubenet=true --name aad-pod-identity --namespace kube-system --version 3.0.3
```
Comment (ferantivero) [inline deployment.md:182]: just out of curiosity, have we found an issue with `v4`?

**Review by ferantivero — APPROVED** [review]
LGTM :rocket:

File: deployment.md (hunk 1 above)
Comment (ferantivero) [reply deployment.md:182]: discard this comment, I found the answer from #207

**Review by ckittel — APPROVED** [review]
Thanks!

**Comment by ckittel** [comment]
@neilpeterson requested that we merge instead of him, so doing that now.  Thanks Neil!

**Pull request merged**